src/workspaces/list/NoContentMessage.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { screen } from '@testing-library/react';
+import React from 'react';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+import { defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+import { NoWorkspacesMessage } from 'src/workspaces/common/NoWorkspacesMessage';
+import { CategorizedWorkspaces } from 'src/workspaces/list/CategorizedWorkspaces';
+import { NoContentMessage } from 'src/workspaces/list/NoContentMessage';
+import { WorkspaceFilterValues } from 'src/workspaces/list/WorkspaceFilters';
+
+jest.mock('src/workspaces/common/NoWorkspacesMessage');
+
+describe('NoContentMessage', () => {
+  const workspaces: CategorizedWorkspaces = {
+    myWorkspaces: [defaultGoogleWorkspace],
+    newAndInteresting: [],
+    featured: [],
+    public: [],
+  };
+
+  const emptyWorkspaces: CategorizedWorkspaces = {
+    myWorkspaces: [],
+    newAndInteresting: [],
+    featured: [],
+    public: [],
+  };
+
+  const filters: WorkspaceFilterValues = {
+    keywordFilter: '',
+    accessLevels: [],
+    tab: 'myWorkspaces',
+    tags: [],
+  };
+
+  it('displays loading message if workspaces are loading', () => {
+    // Arrange
+    const props = { loadingWorkspaces: true, workspaces, filters };
+
+    // Act
+    render(<NoContentMessage {...props} />);
+
+    // Assert
+    expect(screen.getByText('Loading...')).toBeInTheDocument();
+  });
+
+  it('displays no workspaces message if there are no workspaces', () => {
+    // Arrange
+    const props = { loadingWorkspaces: false, workspaces: emptyWorkspaces, filters };
+    asMockedFn(NoWorkspacesMessage).mockReturnValue(<div>mockNoWorkspacesMessage</div>);
```

Comment:
I would be tempted to not mock `NoWorkspacesMessage` and make sure that there is an element with role button and text "Create a New Workspace" in this case.

It is true that what you have here is a stricter unit test, but unless we want to write explicit coverage for `NoWorkspacesMessage` (which seems like overkill), it would be nice to have coverage of what actually displays.